### PR TITLE
🏗️✨ introduce the Nu Html Checker (v.Nu)

### PR DESCRIPTION
**Pull request by DerekNonGeneric**
To validate HTML, this PR introduces the [Nu Html Checker (v.Nu)](https://github.com/validator/validator) in an independent task.

It corrects ~~all~~ [_most_ of the] warnings and [_all_ of the] errors flagged by it.

This is another step in working towards the goal of [keeping the site up-to-date](https://github.com/tc39/tc39.github.io/issues/199).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,11 +21,15 @@
...
+    "node": "^14.8"
+  },
   "devDependencies": {
     "prettier": "^1.17.1",
     "stylelint": "^10.0.1",
     "stylelint-config-prettier": "^5.2.0",
     "stylelint-config-standard": "^18.3.0",
-    "stylelint-order": "^3.0.0"
+    "stylelint-order": "^3.0.0",
+    "vnu-jar": "20.6.30"
```
Comment (DerekNonGeneric) [inline package.json:33]: I'm big on pinning dependencies, but this could also be a caret range.

```suggestion
    "vnu-jar": "^20.6.30"
```

**Review by DerekNonGeneric — COMMENTED** [review]
A few comments to help guide future review.

**Comment by codehag** [comment]
In the future, you don't need to make every modification a separate commit. It is clear enough to have one commit be adding the tool, and the second commit be the changes recommended by the tool.

**Comment by DerekNonGeneric** [comment]
@codehag, I've squashed all of the corrections in into https://github.com/tc39/tc39.github.io/pull/209/commits/ffb1328192eedb0adb02e06207ce3983f5b04006. This is ready to merge, sorry for the delay.

**Comment by DerekNonGeneric** [comment]
I've squashed all of the corrections into https://github.com/tc39/tc39.github.io/pull/209/commits/ffb1328192eedb0adb02e06207ce3983f5b04006. This is ready to merge, sorry for the delay.

**Comment by DerekNonGeneric** [comment]
> In the future, you don't need to make every modification a separate commit.

Sounds good, I was just concerned that it would be difficult to review. Apologies.

**Comment by DerekNonGeneric** [comment]
This PR is good to go and will help a great deal with simplifying #211 by eliminated duplicate effort.

We should also have this merged before https://github.com/tc39/tc39.github.io/pull/213 to ensure the runtime is set up for it.

**Pull request merged**